src/pages/home/library/Mint.tsx

```diff
@@ -0,0 +1,451 @@
+                      CREDITS TO SHOW ON SONG DETAIL
+                    </Typography>
+                    { futureCreditors.map((owner) => (
+                      <Stack
+                        key={ owner.email }
+                        sx={ {
+                          flexDirection: "row",
+                          justifyContent: "space-between",
+                          mt: 2,
+                        } }
+                      >
+                        <Stack>
+                          <Typography>{ `${owner.firstName} ${owner.lastName}` }</Typography>
+                          <Typography variant="subtitle1">
+                            { owner.email }
+                          </Typography>
+                        </Stack>
+                        <Button
+                          color="white"
+                          sx={ { ml: 3 } }
+                          variant="outlined"
+                          width="icon"
+                          onClick={ () => {
+                            setFutureCreditors(
+                              futureCreditors.filter(
+                                (futureOwner) =>
+                                  futureOwner.email !== owner.email
+                              )
+                            );
+                          } }
+                        >
+                          <CloseIcon sx={ { color: theme.colors.white } } />
+                        </Button>
+                      </Stack>
+                    )) }
+
+                    <HorizontalLine sx={ { my: 4 } } />
+                  </>
+                ) : null }
+                <Stack>
+                  <Button
+                    color="white"
+                    variant="outlined"
+                    width="full"
+                    onClick={ () => {
+                      setIsModalOpen(true);
+                    } }
+                  >
+                    Add new owner
+                  </Button>
+                </Stack>
+              </>
+            ) }
+          </Box>
+
+          <Stack direction="row" columnGap={ 2 } mt={ 5 }>
+            <Button
+              onClick={ () => navigate(-1) }
+              variant="secondary"
+              width={
+                windowWidth && windowWidth > theme.breakpoints.values.md
+                  ? "compact"
+                  : "default"
+              }
+            >
+              Cancel
+            </Button>
+
+            { isMintSwitchOn && (
+              <Button
+                disabled={ !isFullyOwned }
+                onClick={ () => {
+                  setIsFirstStep(false);
+                } }
+                width={
+                  windowWidth && windowWidth > theme.breakpoints.values.md
+                    ? "compact"
+                    : "default"
+                }
+              >
+                Next
+              </Button>
+            ) }
```

Comment:
Should this button also be disabled if Credits is missing, is it proper to mint a song without a Credits to show on `song detail`? Or pop up an alert, info, or field requirement warning? Also is song detail the `song description ` on the `INFO` tab? Or is song detail just a minting field?

Reply:
>is it proper to mint a song without a Credits to show on song detail?
That right there is a great question, i will ask product and update this accordingly.
>is song detail the song description  on the INFO tab? Or is song detail just a minting field?
This is something that @scandycuz might have more info on. I would assume the song detail and credit song details are 2 separate things? idk though

Reply:
I guess looking off of that Danketsu metadata I'm still not sure how everything lines up, might just be a point of education for me on the user interface to minting metadata side, or might be part of an existing MVP ticket

the current `ROLE` list doesn't include `Artist` and `Contributing_Artists` as listed on that metadata

Reply:
@dmkirshon Confirmed that a song can be minted without credits. will keep as is 🤷🏽‍♂️

Reply:
Sounds good, just seems a little strange to leave out for metadata unless the artist wants complete anonymity I guess.